src/EnergyPlus/HeatingCoils.cc

```diff
@@ -2342,9 +2342,13 @@ namespace HeatingCoils {
                     OutletAirHumRat = FullLoadOutAirHumRat;
                     OutletAirTemp = FullLoadOutAirTemp;
                 } else {
-                    OutletAirEnthalpy = PartLoadRat * FullLoadOutAirEnth + (1.0 - PartLoadRat) * InletAirEnthalpy;
-                    OutletAirHumRat = PartLoadRat * FullLoadOutAirHumRat + (1.0 - PartLoadRat) * InletAirHumRat;
-                    OutletAirTemp = PartLoadRat * FullLoadOutAirTemp + (1.0 - PartLoadRat) * InletAirDryBulbTemp;
+                    OutletAirEnthalpy =
```

Comment:
It looks like this is protected above for nonzero AirMassFlow variable, but I don't see in this GitHub diff exactly where it is protected from zero HeatingCoil(CoilNum).InletAirMassFlowRate.  I'll need to check it out locally.